app/views/service_providers/show.html.erb

```diff
@@ -100,6 +100,12 @@
         <%= image_tag service_provider.active? ? 'img/alerts/success.svg' : 'img/alerts/error.svg', height: '27', width: '27', :class =>'margin-bottom-neg-105' %>
       </td>
     </tr>
+    <tr>
+      <th scope="row">Service Provider config as YAML</th>
+      <td>
+        <%= render 'service_provider_yaml' %>
+      </td>
+    </tr>
```

Comment:
Let's restrict this to only admins. Regular users won't want or need to see this.

Reply:
Woops! I had done this and accidentally removed the restriction when I put this in a partial 😅